src/components/ContentOpenWith/IntegrationPortal.js

```diff
@@ -24,7 +24,15 @@ class IntegrationPortal extends PureComponent<Props> {
         super(props);
 
         const { integrationWindow }: Props = this.props;
-        this.containerElement = integrationWindow.document.createElement('div');
+        const existingContainer = integrationWindow.document.querySelector(
+            'div',
+        );
+
+        // If the integration window isn't closed, we should reuse the container div and avoid
+        // creating another one.
+        this.containerElement =
```

Comment:
@jstoffan I'm creating a div here but actually manipulating it in `componentDidMount`. This is because render references the container Element so it needs to be created first.